pkg/build/kube/zypperbits.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kube
+
+import (
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// the packages repo
+	repo = "https://download.opensuse.org/tumbleweed/repo/oss/"
+
+	// packages necessary to install
+	packages = "kubernetes-kubelet kubernetes-kubeadm kubernetes-client"
+)
+
+// ZypperBits implements Bits for the official upstream openSUSE packages
+type ZypperBits struct {
+}
```

Comment:
please move `}` to the previous line: `{}`

Reply:
This is [direct translation](https://github.com/kubernetes-sigs/kind/blob/3a6fe2fee6256318e59b1edfe04264942c68894c/pkg/build/kube/aptbits.go#L27-L28) of the `apt` installer, but I will change it.